po/pt_BR.po

```diff
@@ -0,0 +1,453 @@
+msgstr ""
+"Quando marcada, as teclas F1..F12 vão acionar sua função especial,\n"
+"e você precisa manter pressionada a tecla FN para acionar sua função padrão."
+
+#: lib/logitech_receiver/settings_templates.py:87
+msgid ""
+"When unset, the F1..F12 keys will activate their standard function,\n"
+"and you must hold the FN key to activate their special function."
+msgstr ""
+"Quando desmarcada, as teclas F1..F12 vão acionar sua função padrão,\n"
+"e você precisa pressionar a tecla FN para acionar sua função especial."
+
+#: lib/logitech_receiver/settings_templates.py:89
+msgid "Hand Detection"
+msgstr "Detecção de Mão"
+
+#: lib/logitech_receiver/settings_templates.py:90
+msgid "Turn on illumination when the hands hover over the keyboard."
+msgstr "Ligar a iluminação quando a mão estiver sobre o telcado."
+
+#: lib/logitech_receiver/status.py:98
+msgid "No paired devices."
+msgstr "Nenhum dispositivo emparelhado."
+
+#: lib/logitech_receiver/status.py:99
+msgid "1 paired device."
+msgstr "1 dispositivo emparelhado."
+
+#: lib/logitech_receiver/status.py:100
+msgid " paired devices."
+msgstr " dispositivos emparelhados."
+
+#: lib/logitech_receiver/status.py:149 lib/logitech_receiver/status.py:151
+#: lib/logitech_receiver/status.py:203 lib/logitech_receiver/status.py:205
+#: lib/solaar/ui/window.py:143
+msgid "Battery"
+msgstr "Bateria"
+
+#: lib/logitech_receiver/status.py:162 lib/solaar/ui/window.py:150
+msgid "Lighting"
+msgstr "Iluminação"
+
+#: lib/logitech_receiver/status.py:162 lib/solaar/ui/window.py:663
+msgid "lux"
+msgstr "lux"
+
+#: lib/solaar/listener.py:95
+msgid "The receiver was unplugged."
+msgstr "O receptor foi desplugado."
+
+#: lib/solaar/ui/__init__.py:48
+msgid "Permissions error"
+msgstr "Erro de permissão"
+
+#: lib/solaar/ui/__init__.py:49
+#, python-format
+msgid "Found a Logitech Receiver (%s), but did not have permission to open it."
+msgstr "Foi encontrado um Receptor da Logitech (%s), mas sem permissão para abri-lo."
+
+#: lib/solaar/ui/__init__.py:51
+msgid ""
+"If you've just installed Solaar, try removing the receiver and plugging it "
+"back in."
+msgstr ""
+"Se você acabou de instalar o Solaar, tente retirar o receptor e plugá-lo "
+"novamente."
+
+#: lib/solaar/ui/__init__.py:53
+msgid "Unpairing failed"
+msgstr "Desemparelhamento falhou"
+
+#: lib/solaar/ui/__init__.py:54
+#, python-brace-format
+msgid "Failed to unpair %{device} from %{receiver}."
+msgstr "Falha ao desemparelhar %{device} do %{receiver}."
+
+#: lib/solaar/ui/__init__.py:56
+msgid "The receiver returned an error, with no further details."
+msgstr "O receptor retornou um erro, sem maiores detalhes."
+
+#: lib/solaar/ui/about.py:39
+msgid ""
+"Shows status of devices connected\n"
+"through wireless Logitech receivers."
+msgstr ""
+"Mostra o status dos dispositivos conectados\n"
+"através de receptores sem fio da Logitech."
+
+#: lib/solaar/ui/about.py:48
+msgid "GUI design"
+msgstr "Design da GUI"
```

Comment:
Maybe, if the modal has enough space, it is better to be less technical targeting no tech end users.

Something like "Design da interface gráfica"